indico/modules/rb/templates/emails/reservations/reminders/upcoming_occurrence.html

```diff
@@ -30,14 +30,18 @@
     {% set reservation = occurrence.reservation -%}
 
     <strong>{{ occurrence.start_dt | format_date('EEEE dd/MM/yyyy', locale='en_GB') }}</strong> (<strong>{{ _render_time(occurrence) }}</strong>)<br>
-    Room: <strong>{{ reservation.room.full_name }}</strong><br>
-    Reason: <strong>{{ reservation.booking_reason }}</strong><br>
+    {% trans %}Room:{% endtrans %} <strong>{{ reservation.room.full_name }}</strong><br>
+    {% trans %}Reason:{% endtrans %} <strong>{{ reservation.booking_reason }}</strong><br>
     <br>
     {%- if reservation.repeat_frequency.name == 'NEVER' -%}
-        If you do not need to use the room, please
-        <a href="{{ reservation.external_details_url }}">cancel the reservation in Indico</a>.
+        {% trans a=' <a href="{{ reservation.external_details_url }}">'|safe enda='</a>'|safe %}
+            If you do not need to use the room, please
+            {{ a }}cancel the reservation in Indico{{ enda }}.
+        {%- endtrans -%}
     {%- else -%}
-        If you do not need to use the room, please
-        <a href="{{ occurrence.external_cancellation_url }}">cancel the occurrence in Indico</a>.
+        {% trans a=' <a href="{{ occurrence.external_cancellation_url }}">'|safe enda='</a>'|safe %}
```

Comment:
I'd use link/endlink to make it less confusing for translators. Is the leading whitespace needed BTW? The linebreak should already cause a whitespace.